taccsite_cms/settings_custom.example.py

```diff
@@ -79,49 +79,48 @@
 }
```

Comment:
**Note.** The changes on this page are all just example settings. Near identical changes were made to Texascale settings, and the new content is used almost verbatim in ECEP settings, in related PR https://github.com/TACC/Core-CMS-Resources/pull/127.